### PR TITLE
Advanced Engineering Gear ManuDrive.

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
**This pr is a branch of #5952 and should not be merged without that being merged first**

Oh yeah btw, please comment suggestions for things to add, I dont want to put an input wanted thing cause I already made a forum post but yeah.

This pr adds a ManuDrive that spawns in the CE locker. This manudrive contains the recipes for heavy firesuits, at the cost of 10 filbrilith and 6 Met-3. It also contains Industrial Engine Armor, a variant of industrial armor more suited for singularity and TeG work, trading the explosion resistance of a normal indy suit for more fire resistance. This suit costs 17 Met-3, 5 Con-2 and 3 filbrilith. The recipes for RCD's and all there charges are also located in this disk.

## Why's this needed? 
Right now mining does not have many ways to help jobs outside of robotics and cargo. This pr (and other sister prs that will come out later) intend to change the fact. By giving more jobs usage for mining mats, miners supplying rockbox will help the station out even more.

## Changelog

```changelog
(u)Ikea
(*)The CE's locker now contains a ManuDrive containing the recipes for several advanced engineering equipment.
(+)Mining and general fabricators no longer contain RCD related recipes when hacked.
```

### CODE REVIEW

File: code/datums/gamemodes/spy_theft.dm [hunk 1]
```diff
@@ -397,6 +397,7 @@
 	station_bounties[/obj/item/pen/crayon/golden] = 2
 	station_bounties[/obj/item/remote/porter/port_a_sci] = 2
 	station_bounties[/obj/item/clothing/suit/hosmedal] = 3
+	station_bounties[obj/item/manudrive/eng] = 2
```
Comment (BatElite) [inline code/datums/gamemodes/spy_theft.dm:400]: Linter is angry about the / missing in front

Comment (ThatFiveGuys) [reply code/datums/gamemodes/spy_theft.dm:400]: Ah fuck me thought I fixed that, was tired when I made this pr so its not suprising theres some sloppy mistakes.

File: code/obj/item/clothing/helmets.dm [hunk 2]
```diff
@@ -737,6 +737,16 @@
...
 			setProperty("meleeprot_head", 7)
 
+	eng
+		name = "industrial engine helmet"
+		desc = "Goes with Industrial Engineering Armor. Now with zesty citrus-scented visor!"
+		icon_state = "induseng"
+		item_state = "induseng"
+		setupProperties()
+			..()
+			setProperty("exploprot", 0)
```
Comment (Tarmunora) [inline code/obj/item/clothing/helmets.dm:747]: `delProperty` preferred here

File: code/obj/item/clothing/suits.dm [hunk 3]
```diff
@@ -1337,6 +1337,15 @@
...
 			icon_state = "syndie_specialist-heavy"
 			item_state = "syndie_specialist-heavy"
+	eng
+		name = "industrial engine armor"
+		desc = "Very heavy armour for prolonged engine activity. Protects from radiation and fire."
+		icon_state = "induseng"
+		item_state = "induseng"
+		setupProperties()
+			..()
+			setProperty("exploprot", 0)
```
Comment (Tarmunora) [inline code/obj/item/clothing/suits.dm:1347]: `delProperty` here too

File: code/obj/item/manudrive.dm [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+obj/item/manudrive/eng
+	name = "Engineering ManuDrive: Advanced Engineering Equipment"
+	icon_state = "manudriveeng"
+	temp_recipe_string = list(/datum/manufacture/heavy_firesuit,
+	/datum/manufacture/indus_eng,
+	/datum/manufacture/RCD,
+	/datum/manufacture/RCDammo,
+	/datum/manufacture/RCDammomedium,
+	/datum/manufacture/RCDammolarge)
```
Comment (Tarmunora) [inline code/obj/item/manudrive.dm:23]: Remove fab datums from hacked mining fab?

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Review by Tarmunora — APPROVED** [review]

**Comment by ThatFiveGuys** [comment]
Ill fix merge conflicts if and or when #5952 goes through

**Comment by ThatFiveGuys** [comment]
I was probably going to close this branch anyways to redo it so, ill reopen this but seperate in a few days.

**Pull request closed without merging**